### ISSUE TITLE
Trouble running most games 

### ISSUE THREAD

**Issue by jsaviour84** [teknogods/teknoparrot-issues#7]
Please fill in the following information when creating a new issue:

**TeknoParrot version**:
**Did this issue happen with an older version**:
**Game with issue**:
**Detailed description of issue**:
**Reproduction details**:
**Any other details**:
**Pictures of issue**:

**Comment by jsaviour84** [comment]
So started off using 1.04a and id6 works fine. I tried to use any other game (src,sr3, pretty much all the other games) and they dont run. The command line loads and says the game is resuming but the game never shows up. I recently dled 1.06a and now nothing loads. Command line starts then goes away. Also Src also keeps asking to install and it uses adobe flash which no longer works on win10.. I dont know if this has to do with registry? I tried to use my current code but it said it was bad. So those are the issues im having. Any help would be great
